snow/engine/snowman/transitive_test.go

```diff
@@ -2762,54 +2764,39 @@ func TestEngineBubbleVotesThroughInvalidBlock(t *testing.T) {
 	reqID := new(uint32)
 	sender.SendGetF = func(_ context.Context, inVdr ids.NodeID, requestID uint32, blkID ids.ID) {
 		*reqID = requestID
-		if *asked {
-			t.Fatalf("Asked multiple times")
-		}
-		if blkID != blk1.ID() {
-			t.Fatalf("Expected engine to request blk1")
-		}
-		if inVdr != vdr {
-			t.Fatalf("Expected engine to request blk2 from vdr")
-		}
+		require.False(*asked, "Asked multiple times")
+		require.Equal(blkID, blk1.ID(), "Expected engine to request blk1")
+		require.Equal(inVdr, vdr, "Expected engine to request blk1 from vdr")
```

Comment:
`require.Equal` expects the arguments to be `expected`, `actual` (same below)
```suggestion
		require.Equal(blk1.ID(), blkID, "Expected engine to request blk1")
		require.Equal(vdr, inVdr, "Expected engine to request blk1 from vdr")
```
Also not really sure the descriptions are needed. I feel like they don't add much (and we need to make sure they are accurate)... I'd probably just remove them